pbjson-test/src/lib.rs

```diff
@@ -36,6 +36,16 @@ pub mod test {
         include!(concat!(env!("OUT_DIR"), "/test.duplicate_name.rs"));
         include!(concat!(env!("OUT_DIR"), "/test.duplicate_name.serde.rs"));
     }
+    pub mod escape {
+        include!(concat!(
+            env!("OUT_DIR"),
+            "/test.r#abstract.r#type.escape.rs"
+        ));
+        include!(concat!(
+            env!("OUT_DIR"),
+            "/test.r#abstract.r#type.escape.serde.rs"
+        ));
+    }
```

Comment:
Could we possibly get some tests of serializing to/from these types?

Reply:
[c608e46](https://github.com/influxdata/pbjson/pull/67/commits/c608e46ffe4ea6057d2e5eea9fca479e52640293)